spring-boot-project/spring-boot-docker-compose/src/dockerTest/resources/org/springframework/boot/docker/compose/service/connection/db2/db2-compose.yaml

```diff
@@ -0,0 +1,18 @@
+services:
+  database:
+    image: '{imageName}'
+    ports:
+      - '50000'
+    privileged: true
```

Comment:
I don't think you can run `privileged` containers on the CI.

Reply:
Otherwise `Connection reset` exception always be thrown, that's why testcontainers `Db2Container` need that also.
https://github.com/testcontainers/testcontainers-java/blob/04206d9dff440e875906f01151f09ad04da3c68f/modules/db2/src/main/java/org/testcontainers/containers/Db2Container.java#L60